### PR TITLE
add parameter option to disable git status in zsh and bash prompt

### PR DESCRIPTION
**Pull request by kjhenner**
add parameter option to disable git status in zsh and bash prompt

### CODE REVIEW

File: manifests/bash.pp [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-  $default  = true,
-  $password = undef,
-  $replace  = false,
+  $user      = 'root',
+  $group     = 'root',
+  $homedir   = '/root',
+  $default   = true,
+  $password  = undef,
+  $replace   = false,
+  $gitprompt = false,
```
Comment (binford2k) [inline manifests/bash.pp:8]: the variable name should be consistent unless there's a reason otherwise

File: templates/bashrc.puppet.epp [hunk 2]
```diff
@@ -40,19 +40,10 @@ function git_color ()
     fi
 }
 
-# Call the above functions inside the PS1 declaration
+# Call the above functions inside the PS1 declaration if we want to see git status
+<% if $gitprompt { -%>
 export PS1='\[$(git_color)\]$(parse_git_branch)\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\] \$ '
+<% } -%>
 
-<% if @hostname =~ /learn/ -%>
```
Comment (samuelson) [inline templates/bashrc.puppet.epp:46]: Is all of this quest stuff going away?

Comment (kjhenner) [inline templates/bashrc.puppet.epp:46]: This stuff is changing with the quest tool refactor. I'm working on that in a new branch. It's currently broken with regard to the other repos involved in that build, so deleting it here isn't a regression.

File: manifests/bash.pp (hunk 1 above)
Comment (kjhenner) [inline manifests/bash.pp:8]: Just changed these to be consistent

**Pull request merged**